app/Repositories/PizzaRepository.php

```diff
@@ -3,31 +3,50 @@
 namespace App\Repositories;
 
 use App\Models\Pizzas;
-use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Collection;
 
 class PizzaRepository
 {
-    public static function validatePizzaFields($request)
+
+    public static function getListPizzasInDatabase(): ?Collection
     {
-        if (empty($request['pizza_name']) || empty($request['pizza_price']) || empty($request['pizza_description'])) {
-            return $request->session()->flash('mensagemErro', 'Cadastro errado, preencha os dados!');
-        }
+        return Pizzas::get()->sortByDesc('id');
+
+    }
 
-        Pizzas::create($request->all());
-        $request->session()->flash('mensagem', 'Cadastro realizado com sucesso!');
+    public static function createPizza($request): ?Pizzas
+    {
+        return Pizzas::create($request->all());
 
     }
 
-    public static function validateIfHasPizzaInDatabase($request)
+    public static function deletePizza($request): void
     {
-        $pizzas = DB::table('pizzas')
-        ->orderBy('id', 'desc')
-        ->get();
-        
-        if(empty($pizzas)){
-            $mensagem = $request->session()->get('mensagem');
-            
-        }
-        
+        $pizza = new Pizzas();
+        $pizza->where('id', $request->id)->delete();
```

Comment:
tem jeito melhor de se fazer isso.

```suggestion
        Pizzas::query->where('id', $request->id)->delete();
```